reference/api/tasks.md

```diff
@@ -2,10 +2,172 @@
 
 The `/tasks` route gives information about the progress of [asynchronous operations](/learn/advanced/asynchronous_operations.md).
 
+## Task object
+
+```json
+{
+  "uid": 4,
+  "indexUid" :"movie",
+  "status": "failed",
+  "type": "indexDeletion",
+  "details": {
+    "deletedDocuments": 0
+  },
+    "error": {
+      "message": "Index `movie` not found.",
+      "code": "index_not_found",
+      "type": "invalid_request",
+      "link": "https://docs.meilisearch.com/errors#index_not_found"
+    },
+      "duration": "PT0.001192S",
+      "enqueuedAt": "2022-08-04T12:28:15.159167Z",
+      "startedAt": "2022-08-04T12:28:15.161996Z",
+      "finishedAt": "2022-08-04T12:28:15.163188Z"
+}
+```
+
+### `uid`
+
+**Type**: Integer
+**Description**: Unique sequential identifier of the task
+
 :::note
 The task `uid` is incremented **globally.**
 :::
 
+### `indexUid`
+
+**Type**: String
+**Description**:  Unique identifier of the targeted index
+
+::: note
+This value is always `null` for `dumpCreation` tasks.
+:::
+
+### `status`
+
+**Type**: String
+**Description**: Status of the task. Possible values are `enqueued`, `processing`, `succeeded`, `failed`
+
+### `type`
+
+**Type**: String
+**Description**: Type of operation performed by the task. Possible values are `indexCreation`, `indexUpdate`, `indexDeletion`, `documentAdditionOrUpdate`, `documentDeletion`, `settingsUpdate`, `dumpCreation`
+
+### `details`
+
+**Type**: Object
+**Description**: Detailed information on the task payload. This object's contents depend on the task's `type`
+
+::: note
+The `details` object will only show values for the updated field. For example, if you update the faceting settings, the `details` object will only contain the updated faceting value:
+
+```json
+{
+   "uid": 3,
+   "indexUid": "movies",
+   "status": "succeeded",
+   "type": "settingsUpdate",
+   "details": {
+      "faceting": {
+         "maxValuesPerFacet": 2
+      }
+   },
+   "duration": "PT0.000682S",
+   "enqueuedAt": "2022-08-02T15:41:57.955977Z",
+   "startedAt": "2022-08-02T15:41:57.957002Z",
+   "finishedAt": "2022-08-02T15:41:57.957684Z"
+}
+```
+
+:::
```

Comment:
I don't think this is true. Either way it seems to be more confusing than helpful so I think we should remove it.

```suggestion
```